airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/sensors/gcs.py

```diff
@@ -44,3 +53,66 @@ def new_gcs_blobs_sensor_definition(context: SensorEvaluationContext):
             return RunRequest(run_key=run_key)
 
     return new_gcs_blobs_sensor_definition
+
+def new_gcs_blobs_partition_sensor(
```

Comment:
Added: A new sensor that can generate partition based jobs.

Partition jobs are how we say given an array of metadata file names, generate registry entry assets for each